specs/platform_info.table

```diff
@@ -10,4 +10,7 @@ schema([
     Column("volume_size", INTEGER, "(Optional) size of firmware volume"),
     Column("extra", TEXT, "Platform-specific additional information"),
 ])
+extended_schema(WINDOWS, [
```

Comment:
Do you think this column will have meaning on the other platforms?

Reply:
I'd say it can make sense on Linux probably, but there is no need to prematurely add the column to platforms without an implementation. It can be added when an implementation is ready.